src/transport/SecureSession.cpp

```diff
@@ -20,6 +20,23 @@
 namespace chip {
 namespace Transport {
 
+Access::Subject SecureSession::GetSubject() const
+{
+    if (IsOperationalNodeId(mPeerNodeId))
+    {
+        return Access::Subject::Create<Access::NodeSubject>(GetFabricIndex(), mPeerNodeId);
+    }
+    else if (IsPAKEKeyId(mPeerNodeId))
+    {
+        return Access::Subject::Create<Access::PaseSubject>(GetFabricIndex(), static_cast<uint16_t>(mPeerNodeId >> 48));
```

Comment:
There should not be PAKE subjects anymore (they are now reserved for future use, see https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/4878)

Reply:
Done, use a unique subject to identify all PASE sessions.